monai/bundle/scripts.py

```diff
@@ -417,37 +430,44 @@ def load(
             If used, it should be in the form of "repo_owner/repo_name/release_tag".
         remove_prefix: This argument is used when `source` is "ngc". Currently, all ngc bundles
             have the ``monai_`` prefix, which is not existing in their model zoo contrasts. In order to
-            maintain the consistency between these two sources, remove prefix is necessary.
+            maintain the consistency between these three sources, remove prefix is necessary.
             Therefore, if specified, downloaded folder name will remove the prefix.
         progress: whether to display a progress bar when downloading.
         device: target device of returned weights or module, if `None`, prefer to "cuda" if existing.
         key_in_ckpt: for nested checkpoint like `{"model": XXX, "optimizer": XXX, ...}`, specify the key of model
             weights. if not nested checkpoint, no need to set.
         config_files: extra filenames would be loaded. The argument only works when loading a TorchScript module,
             see `_extra_files` in `torch.jit.load` for more details.
-        net_name: if not `None`, a corresponding network will be instantiated and load the achieved weights.
```

Comment:
May I know why remove this arg and net_kwargs?

Thanks.

Reply:
Hi @Nic-Ma, I replaced it with a "model" arg, so that the user doesn't have to pass the name and args of the net separately, do you think it makes sense?
https://github.com/Project-MONAI/MONAI/blob/6938718b757e0366b8fc6344a1e7dbafcf588bbd/monai/bundle/scripts.py#L413

Reply:
OK, so it's a breaking change, please add a `deprecated` decorator.

Thanks.

Reply:
Thanks, have added it in this [commit](https://github.com/Project-MONAI/MONAI/pull/6835/commits/561a68e362026de630bc247f542eb8bccf852c1a).